services/spar/src/Spar/App.hs

```diff
@@ -1,5 +1,7 @@
 {-# LANGUAGE GeneralizedNewtypeDeriving #-}
 {-# LANGUAGE RecordWildCards #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
Needed for the MonadThrow/Catch instances for Sem. Do you have a better place to put orphans?

Reply:
Not really, but they are a temporary thing, right?